src/vs/platform/extensionManagement/node/extensionSignatureVerificationService.ts

```diff
@@ -26,7 +27,7 @@ export interface IExtensionSignatureVerificationService {
 }
 
 declare module vsceSign {
-	export function verify(vsixFilePath: string, signatureArchiveFilePath: string): Promise<boolean>;
+	export function verify(vsixFilePath: string, signatureArchiveFilePath: string, logger: ILogger): Promise<boolean>;
```

Comment:
I think we shall not pass the caller types to the api. It shall be otherway around. I mean at present Logger is defined by the caller and not by the api. Also passing caller objects as is not good. I would recommed node-vsce-sign module to define the logger API and let the caller uses it.